rosdep/python.yaml

```diff
@@ -3633,6 +3633,13 @@ python-pysnmp:
   fedora: [pysnmp]
   gentoo: [dev-python/pysnmp]
   ubuntu: [python-pysnmp4]
+python-pystemd:
```

Comment:
Confusingly, while I understand this is meant to be used with Python 2, the contributing guide says this key should be named [`python3-systemd-pip`](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#pip-1).

Maybe this should be named `python-systemd-pip` instead? @clalancette thoughts?

Reply:
That is confusing indeed!

Happy to convert to `python-systemd-pip`

Reply:
Please convert it to `python3-systemd-pip` The name will not effect which version of python it uses. But as we look towards the future all default platforms will be using python3* and so we want it to be sorted into the right place.